client/src/App.jsx

```diff
@@ -8,10 +8,10 @@ import { InMemoryCache } from 'apollo-cache-inmemory';
 import { setContext } from 'apollo-link-context';
 import { onError } from 'apollo-link-error';
 import { createUploadLink } from 'apollo-upload-client';
-
+import { MuiPickersUtilsProvider } from '@material-ui/pickers';
 import { withCookies, Cookies } from 'react-cookie';
 import moment from 'moment';
-
+import MomentUtils from '@date-io/moment';
```

Comment:
Same case here, check for its call in home.jsx file in the current state of the develop branch

Reply:
You are right, I fixed in https://github.com/sjdonado/quevent/commit/b21d0ee6cbaaf221f859d6c365bd37f68f0ad404